### PR TITLE
Refactoring(structure): move case fields from sample to case

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
### Changed
- move case fields _synopsis_ and _cohorts_ from sample to case

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] tested in https://github.com/Clinical-Genomics/cgweb/pull/416

## Review
- [x] code approved by VJ
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] in https://github.com/Clinical-Genomics/cgweb/pull/416

### CODE REVIEW

File: cg/apps/orderform/orderform_parser.py [hunk 1]
```diff
@@ -81,24 +81,37 @@ def get_pools(self) -> List[OrderPool]:
...
         for sample in case_samples:
             if not sample.panels:
                 continue
             gene_panels.update(set(sample.panels))
 
+        cohorts = set()
+        for sample in case_samples:
+            if not sample.cohorts:
+                continue
+            cohorts.update(set(sample.cohorts))
```
Comment (Vince-janv) [inline cg/apps/orderform/orderform_parser.py:106]: If samples have different cohorts these are just added? Why is this treated differently compared to synopsis?

**Review by Vince-janv — APPROVED** [review]
Nice and clean! 🥇

**Review by Vince-janv — APPROVED** [review]
Super! 👍

**Comment by patrikgrenfeldt** [comment]
Deployed to production:
image

**Pull request merged**